core-web/libs/edit-content/src/lib/fields/dot-edit-content-host-folder-field/dot-edit-content-host-folder-field.component.html

```diff
@@ -0,0 +1,18 @@
+<p-treeSelect
+    [formControl]="pathControl"
+    [options]="options()"
+    [attr.aria-labelledby]="'field-' + field.variable"
+    (onNodeSelect)="onNodeSelect($event)"
+    (onNodeExpand)="onNodeExpand($event)"
+    selectionMode="single">
+    <ng-template let-node pTemplate="default">
+        <span>{{ node.label | truncatePath }}</span>
+    </ng-template>
+    <ng-template pTemplate="triggericon">
+        @if (sitesStatus() === 'loading') {
+            <i class="pi pi-spin pi-spinner"></i>
+        } @else {
+            <i class="pi pi-chevron-down"></i>
+        }
```

Comment:
You can use a computed/signal here to set the classes.

```html
<I [ngClass]="iconClasses()"></i>
```

Reply:
good.

Reply:
done!